### PR TITLE
Bug 1826339: vendor: bump our k8s vendor

### PR DESCRIPTION
**Pull request by squeed**
This is to pick up the backport of upstream 90175.

It also seems to fix some weirdness with go.mod that I don't really understand, but go mod tidy is happy.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -68,30 +67,30 @@ replace (
...
-	k8s.io/cri-api => github.com/openshift/kubernetes/staging/src/k8s.io/cri-api v0.0.0-20200506130333-d411faf4483c
-	k8s.io/csi-translation-lib => github.com/openshift/kubernetes/staging/src/k8s.io/csi-translation-lib v0.0.0-20200506130333-d411faf4483c
-	k8s.io/kube-aggregator => github.com/openshift/kubernetes/staging/src/k8s.io/kube-aggregator v0.0.0-20200506130333-d411faf4483c
-	k8s.io/kube-controller-manager => github.com/openshift/kubernetes/staging/src/k8s.io/kube-controller-manager v0.0.0-20200506130333-d411faf4483c
-	k8s.io/kube-proxy => github.com/openshift/kubernetes/staging/src/k8s.io/kube-proxy v0.0.0-20200506130333-d411faf4483c
-	k8s.io/kube-scheduler => github.com/openshift/kubernetes/staging/src/k8s.io/kube-scheduler v0.0.0-20200506130333-d411faf4483c
-	k8s.io/kubectl => github.com/openshift/kubernetes/staging/src/k8s.io/kubectl v0.0.0-20200506130333-d411faf4483c
-	k8s.io/kubelet => github.com/openshift/kubernetes/staging/src/k8s.io/kubelet v0.0.0-20200506130333-d411faf4483c
-	k8s.io/kubernetes => github.com/openshift/kubernetes v1.5.0-alpha.0.0.20200504100644-ad077cd883ae
```
Comment (squeed) [inline go.mod:89]: I'm not sure how this snuck in, yikes...

**Comment by danwinship** [comment]
oops, I was thinking that as long as the kube PR merged first the sdn PR would automatically pick it up but obviously it doesn't actually work that way...

**Pull request merged**